index.rst

```diff
@@ -1,13 +1,15 @@
 Welcome to the UBports documentation!
 =====================================
 
-Welcome to the official documentation of the UBports project!
+Welcome to the official documentation of the UBports project! We are a team of engineers, designers, community managers, translators, students and volunteers who together bring to life Ubuntu Touch, a mobile operating system focused on ease of use, privacy, and convergence. We are also lovers of Free software and open collaboration. You can find us all over the web, but Github provides the best starting point for planning your involvement. If you are new, head to get.ubuntu.com to get started
 
-UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
+Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence. It features a tablet-oriented user interface with access to a desktop interface when a keyboard or mouse is connected.
 
 On this website you find guides to :doc:`install Ubuntu Touch on your mobile phone <userguide/install>`, :doc:`use Ubuntu Touch <userguide/dailyuse/index>`, :doc:`develop Ubuntu Touch apps <appdev/index>`, :doc:`port Ubuntu Touch to an Android handset <porting/introduction/index>` and :doc:`learn more about system components <systemdev/index>`. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
 
-If you want to help improving this documentation, :doc:`the Documentation contribute page <contribute/documentation>` will get you started.
+If you want to help improving this documentation you can, in many different ways including: Fork this repository.  Make progress from where I've stalled with updates/corrections/additions from others from the Issues Todo list.  Submit a document translation in a different language. :doc:`the Documentation contribute page <contribute/documentation>` will get you started.
+
+If you would like to get feedback before you commit, you may join UBports Doc Group @ubports_docs on telegram.
```

Comment:
Do we really want to have all group links on the internet? That might attract more spam than contributors? :thinking: 

I think you generally get feedback AFTER committing in the review process, don't you?

Reply:
Oh so you ok help users on 'Welcome group' but now willing to get help from random writers?

Reply:
Not welcoming writers to a group is a terrible decision! I talked yesterday with Matero, he pointed some deep-archived of design content ready to go Docs. That no one know about it. If it will buried, no one use it. True technical writers should pick it up and turn it to a piece.

Reply:
To start writing you don't NEED to be member of the writers group. You can create a PR and go from there. Also content is more static. It is sometimes easier to comment on a PR (as we do here) than have a lengthy discussion in Telegram.

If you have trouble installing, you need some live help. A PR or issue on git is not as (instantly) helpful as a telegram group. So there is some difference.

Regarding Q&A group, Yes, 5 or so testers joined. But also about twenty spambots joined. So I think some groups are fine to be on telegram because we want people to find them, coping with the spambots. But not all need to be there. I agree that deciding which to keep where is not always easy.

Reply:
So let's point where 'what' should happen. Like at KDE :)
Group for Ideas and teaming and writing for reviewing and publishing .. idk

Reply:
I found few  'Jems' where members started initiatives and failed due to collaborations or did knew to do next.